spec/system/supervisors/editing_supervisor_reports_spec.rb

```diff
@@ -28,9 +27,8 @@
     check 'I can completely discount the operator' # But don't give a reason
 
     click_button 'Save supervisor report'
-    wait_for_ajax!
 
-    expect(page.current_path).to eq supervisor_report_path(incident.supervisor_report)
+    expect(page).to have_current_path supervisor_report_path(incident.supervisor_report)
```

Comment:
Here is an example that _would_ have failed. `have_current_path` waits, but compairing via `eq` wouldn't.